### ISSUE TITLE
Why is training loss smaller than validation loss or test loss in univariate setting?

### ISSUE THREAD

**Issue by PhoebeChen123** [zhouhaoyi/Informer2020#283]
It is normal in multivariate setting

**Comment by zhouhaoyi** [comment]
The training loss typically will be smaller than the test loss. But the comparison with validation loss may be subject to change.
